src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,38 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+import java.util.function.BiFunction
+
+enum class Operator(
+    private val operator: String,
+    private val expression: BiFunction<Int, Int, Int>
+) {
+    PLUS(
+        operator = "+",
+        expression = BiFunction { num1, num2 -> num1 + num2 }
+    ),
+    MINUS(
+        operator = "-",
+        expression = BiFunction { num1, num2 -> num1 - num2 }
+    ),
+    MULTIPLY(
+        operator = "*",
+        expression = BiFunction { num1, num2 -> num1 * num2 }
+    ),
+    DIVIDE(
+        operator = "/",
+        expression = BiFunction { num1, num2 -> num1 / num2 }
+    ),
+    ;
+
+    companion object {
+        fun findByStringOperator(operator: String): Operator {
+            return Operator.values()
+                .find { it.operator == operator } ?: throw IllegalArgumentException("문자열 계산기에서 지원하지 않는 연산자에요.")
```

Comment:
Operator의 내부 함수를 Operator 내부에서 사용하는 것이기 때문에 아래와 같이 `Operator`는 제거해도 될 것 같아요. 😃 

```suggestion
            return values().find { it.operator == operator } ?: throw IllegalArgumentException("문자열 계산기에서 지원하지 않는 연산자에요.")
```